config/parser.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// package config implements the configuration Parser.
+// Package config implements the configuration Parser.
```

Comment:
This comment is no longer correct.

Reply:
Removed (config package is already documented)